website/views.py

```diff
@@ -167,11 +167,27 @@ def get_orders():
 @views.route('student/place-order', methods=['POST'])
 @login_required
 def order():
-    data = request.get_json()
-    status = place_order(data)  
-    if status:
+    try:
+        data = request.get_json()
+        place_order(data)  
         return jsonify({'message': 'Order placed successfully'}), 201
-    else:
+    except Exception as e:
+        logging.exception(e)
         return jsonify({'message': 'Sorry! Unexpected error occured while placing the order'}), 500
 
 
+
+@views.route('student/cancel', methods=['PUT'])
+@login_required
+def cancel():
+    try:
+        order_id = request.get_json().get('order_id')
+        print("OID:"+str(order_id))
```

Comment:
Is this ``` print ``` statement necessary? If it is, You can use logger to print statement .

Reply:
That was used for testing purpose. It is now removed in the latest commit.